### ISSUE TITLE
Document Telegraf setup for collecting metrics from AWS services

### ISSUE THREAD

**Issue by gunnaraasen** [influxdata/docs.influxdata.com#2314]
Some users find it hard to discover which AWS services can be monitored by Telegraf and how to configure that monitoring.

Metrics from AWS services can be collected via the [AWS CloudWatch plugin](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/cloudwatch) in most cases or by setting up a file plugin to ingest S3 logs in some other cases (ELB logs, DynamoDB). The final configuration is not obvious. I'd like to suggest that we add the following to the Telegraf docs.

- Separate configuration pages for AWS services that explain how to set up monitoring of those services, e.g. a Telegraf config and examples of AWS IAM permissions required and the metrics returned when monitoring different services.
- An overview page for AWS monitoring using Telegraf that lists all the AWS services that can be monitored by Telegraf.
- AWS service configuration pages should also be included in the [main plugin list](http://docs.influxdata.com/telegraf/v1.11/plugins/plugin-list/).

###### URL for relevant page?
I think we should use Datadog's AWS page as an example of which AWS services Telegraf can monitor. `https://www.datadoghq.com/aws-monitoring/`

###### What products and version are you using?
Telegraf

###### Where did you look before opening the issue?
(Telegraf plugin list)[http://docs.influxdata.com/telegraf/v1.11/plugins/plugin-list/]

We may also want to link to dashboards for each service when available.

**Comment by danielnelson** [comment]
We have something similar to this for the Jolokia plugin, which can also be quite hard to configure. https://github.com/influxdata/telegraf/tree/master/plugins/inputs/jolokia2/examples

Perhaps we can generalize this pattern and hook it into docs.influxdata.com.

**Comment by gunnaraasen** [comment]
I agree there’s definitely value in generalizing the pattern. There are a bunch of plugins where specific configurations for different services would be helpful, especially for anything that requires parsing logs or JSON messages.

**Comment by rawkode** [comment]
I need to do this for billing information on AWS, so maybe I can help @pierwill @noramullen1
